translate_zh_simple.py

```
@@ -0,0 +1,1573 @@
+    result = {}
+    for key, value in d.items():
+        if isinstance(value, dict):
+            result[key] = translate_dict(value)
+        elif isinstance(value, str):
+            result[key] = translate_text(value)
+        else:
+            result[key] = value
+    return result
+
+# Translate zh.json
+zh_translated = translate_dict(zh)
```

Ruff (0.15.18)

[error] Undefined name `zh`

(F821)